### PR TITLE
PEP8 conventions and Issue #24

### PR DESCRIPTION
**Pull request by TrustedMercury**
- Updates code to follow PEP8 Conventions
- Resolves [Issue #24](https://github.com/woodgern/confusables/issues/24) by updating confusables.txt
- Added PEP8 Linter Badge to README.md

### CODE REVIEW

File: confusables/__init__.py [hunk 1]
```diff
@@ -42,17 +43,19 @@ def confusable_characters(char):
         return [char]
     return None
 
+
 def confusable_regex(string, include_character_padding=False):
-    space_regex = "[\*_~|`\-\.]*" if include_character_padding else ''
+    space_regex = r"[\*_~|`\-\.]*" if include_character_padding else ''
```
Comment (woodgern) [inline confusables/__init__.py]: This isn't needed, and it appears to be outside of the scope of the PR

**Comment by woodgern** [comment]
Will need to run `make parse` as well, since that's what actually builds the confusable mapping. Otherwise, we won't actually know if it breaks the test suite or not

File: confusables/__init__.py (hunk 1 above)
Comment (TrustedMercury) [reply confusables/__init__.py]: Fixed 👍🏽

**Comment by TrustedMercury** [comment]
> Will need to run `make parse` as well, since that's what actually builds the confusable mapping. Otherwise, we won't actually know if it breaks the test suite or not

I'm unable to do so due to some OS decoding errors as I use a Windows system whereas it seems like confusables was built in a Linux system.
